app/templates/index.hbs

```diff
@@ -0,0 +1,12 @@
+
+<h1>remEMBER</h1>
+
+{{#each model as |reminder|}}
```

Comment:
@Tman22 I thought that was the point of the second issue. The first issue asked to have them load in the main route. The second issue asked them to be moved to reminders. Did we misunderstand that?